convert2rhel/pkghandler.py

```diff
@@ -366,6 +369,14 @@ def get_packager(pkg_obj):
     return packager.split("<", 1)[0].rstrip()
 
 
+def get_vendor(pkg_obj):
+    # The vendor may not be set for all packages
+    if hasattr(pkg_obj, "vendor") and pkg_obj.vendor:
```

Comment:
Looks good. I'd use here `try, except` construction. It will look simpler and will work faster

Reply:
Artem brought it up in his review, but Fellipe mentioned vendor attribute may be `None`, meaning try-catch would not work as expected.

Anyhow, PR is already merged ;)